### PR TITLE
[Feral] Add tracking of Ashamane's Rip uptime

### PR DESCRIPTION
**Pull request by Anatta336**
Added a statistic and suggestion for the uptime of Ashamane's Rip from the artifact trait Ashamane's Bite.

I'm a little uncertain what statisticOrder to set for it. It usually provides around 5% of total damage, so have put it as OPTIONAL(0) for now.

### CODE REVIEW

File: src/Parser/Druid/Feral/CHANGELOG.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export default `
+24-01-2018 - Feral Druid: Added Ashamane's Rip uptime tracking.
```
Comment (Gebuz) [inline src/Parser/Druid/Feral/CHANGELOG.js]: Remember to add your name! :D

File: src/Parser/Druid/Feral/Modules/Traits/AshamanesRip.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    
+    this.damage += (event.amount || 0) + (event.absorbed || 0) + (event.overkill || 0);
+  }
+
+  get ashamanesRipUptime() {
+    return this.enemies.getBuffUptime(SPELLS.ASHAMANES_RIP.id) / this.owner.fightDuration;
+  }
+
+  suggestions(when) {
```
Comment (Gebuz) [inline src/Parser/Druid/Feral/Modules/Traits/AshamanesRip.js]: It is prefered that you use the new format for suggestions, you can see an example here:
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/Parser/Druid/Balance/Modules/Features/MoonfireUptime.js

**Comment by Anatta336** [comment]
Thank you for the quick response!

**Comment by Gebuz** [comment]
Thanks!

**Pull request merged**